assets/js/util/tracking/index.test.js

```diff
@@ -78,18 +78,24 @@ describe( 'trackEvent', () => {
 
 		trackEvent( 'category', 'name', 'label', 'value' );
 
-		expect( push ).toHaveBeenCalledWith( [
-			'event',
-			'name',
-			{
-				send_to: config.trackingID,
-				event_category: 'category',
-				event_label: 'label',
-				event_value: 'value',
-				dimension1: 'https://www.example.com',
-				dimension2: 'true',
-				dimension3: config.userIDHash,
-			},
-		] );
+		function makeArguments() {
+			return arguments;
+		}
+		// dataLayerPush must push `arguments` onto the data layer.
```

Comment:
Might be nice to elaborate on this here (or in the test utils file, if you move it).

```suggestion
		// dataLayerPush must push `arguments` onto the data layer.
		// Because `arguments` is a special, `Array`-like object (but
		// not an actual `Array`, we need to create another
		// `Arguments` object, which can only be done using the magic
		// variable `arguments`.
```